### PR TITLE
Upgrade Graphcool Dependencies

### LINKED ISSUE #73: fix security NPM vunerability

### PR DESCRIPTION
**Pull request by fredmon3**
This PR closes #73  (recreated from #118)

#### What does this PR do?
This PR upgrades select packages to eliminate known vulnerabilities.

### CODE REVIEW

**Comment by wdoug** [comment]
@fredmon3 updated `package-lock.json` without updating the `package.json`? Can you explain? Also, how should this be tested?

**Comment by fredmon3** [comment]
@wdoug The graphcool package has a nested hierarchy of caret dependencies.  All I did was reinstall all of graphcool's dependencies (to take the latest caret dependency specified by graphcool for each one.)  This changes the `package-lock.json` but not the actual version of graphcool that we're using in `package.json`.  The point was to get rid of the "tar" < 2.0.0 dependency we had in the `package-lock.json` previously.

File: package-lock.json [hunk 1]
```diff
@@ -3421,33 +2626,100 @@
...
       "dev": true,
       "requires": {
-        "adm-zip": "0.4.7",
-        "ext-name": "1.0.1",
-        "get-stdin": "0.1.0",
-        "mkdirp": "0.3.5",
-        "nopt": "2.2.1",
-        "rimraf": "2.6.1",
-        "stream-combiner": "0.0.4",
-        "tar": "0.1.20",
```
Comment (fredmon3) [inline package-lock.json:3436]: Here is the "tar" dependency in question.

File: package-lock.json [hunk 2]
```diff
@@ -3421,33 +2626,100 @@
       "integrity": "sha1-9lNNFRSCabIDUue+4m9QH5oZEpA="
     },
     "decompress": {
-      "version": "0.2.5",
-      "resolved": "https://registry.npmjs.org/decompress/-/decompress-0.2.5.tgz",
-      "integrity": "sha1-0hMjPv4GbM2A2RTXk/GzDNmEuEc=",
+      "version": "4.2.0",
```
Comment (fredmon3) [inline package-lock.json]: Upgraded "decompress" from 0.2.5 to 4.2.0.

File: package-lock.json [hunk 3]
```diff
@@ -5828,89 +4341,59 @@
...
       "dev": true,
       "requires": {
-        "archiver": "2.1.0",
+        "archiver": "2.1.1",
         "callsites": "2.0.0",
-        "chalk": "2.3.0",
+        "chalk": "2.3.2",
         "chokidar": "1.7.0",
-        "download-github-repo": "0.1.3",
+        "download-github-repo": "0.1.4",
```
Comment (fredmon3) [inline package-lock.json:4348]: This, "download-github-repo," is the "graphcool-cli" dependency that caused the ripple effect removing the offending version of "tar"

**Review by wdoug — APPROVED** [review]
Let's do this!

**Pull request merged**